### ISSUE TITLE
all d3 experiments should have consistent UX

### ISSUE THREAD

**Issue by vanch3d** [vanch3d/nvl-slim#23]
* Narrative and Network graphs to have similar elements such as the popup doc panel
* consistent responsive layout (Narrative scrolling & max size computation quite good)
* maybe using a reusable charting lib like c3
